source/settings/core.rst

```diff
@@ -2098,6 +2117,15 @@ Example:
 
 Controls if "Received:" header should be added to delivered mails.
 
+
+.. _setting-lmtp_address_translate:
+
+``lmtp_address_translate``
+--------------------------
```

Comment:
not a huge issue in docs, but this begins already in one commit too early and is finished in the next one.